docs/working-with-projects/working-with-oci/working-with-helm-oci-charts.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Working with OCI Helm Charts
+weight: 65
+---
+
+With the release of Helm 3.8.0, Helm is able to store and work with charts in container registries, as an alternative to Helm repositories. This feature, which used to be an experimental feature, is now generally available.
+
+Please see Helm documentation on [storing charts in OCI](https://helm.sh/blog/storing-charts-in-oci/).
+
+## Pulling Helm Charts
+
+If the project that the helm chart belongs to is private, you must sign in first:
+
+```sh
+helm registry login <harbor_address>
+```
+
+You can now pull a chart:
+
+```sh
+helm pull oci://<harbor_address>/<project>/<chart_name> --version <version>
+```
+
+e.g.
+```sh
+helm pull oci://demo.goharbor.io/oci/demo --version 0.1.0
```

Comment:
maybe change `oci://demo.goharbor.io/oci/demo` to `oci://demo.goharbor.io/oci/oci-charts` to make it consistent between the `docs/img/oci/oci-chart-main-view.png` image and the text.